### PR TITLE
Readding recipe for CLHEP

### PR DESCRIPTION
**Pull request by giacomov**
Continuation of #4608 . I had to restart from a clean branch of `master` because I messed up the git rebase procedure and imported by error many commits from master in the branch.

### CODE REVIEW

**Comment by giacomov** [comment]
+1

**Comment by Areustle** [comment]
👍

**Comment by giacomov** [comment]
@isuruf this pull request has been idle for months... is there a problem with it? Is there anything else I need to do?

File: recipes/clhep/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake ../CLHEP/ -DCMAKE_SHARED_LINKER_FLAGS="-Wl,-rpath,${PREFIX}/lib" -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}
```
Comment (isuruf) [inline recipes/clhep/build.sh]: You can remove `-DCMAKE_SHARED_LINKER_FLAGS="-Wl,-rpath,${PREFIX}/lib"` and `-DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}`

File: recipes/clhep/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - test -f ${PREFIX}/lib/libCLHEP.so  # [linux]
+    - test -f ${PREFIX}/lib/libCLHEP.dylib  # [osx]
+    - test -d ${PREFIX}/include/CLHEP
+    - conda inspect linkages -p $PREFIX clhep  # [not win]
+    - conda inspect objects -p $PREFIX clhep  # [osx]
+
+about:
+  home: http://proj-clhep.web.cern.ch/proj-clhep/
+  license: GPL
```
Comment (isuruf) [inline recipes/clhep/meta.yaml]: Which version?

**Comment by isuruf** [comment]
To get a review, you need to ping  `@conda-forge/staged-recipes`, otherwise we don't know the PR is ready to be reviewed

**Comment by giacomov** [comment]
@conda-forge/staged-recipes this should be ready to merge.

Thanks!

File: recipes/clhep/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - conda inspect linkages -p $PREFIX clhep  # [not win]
+    - conda inspect objects -p $PREFIX clhep  # [osx]
+
+about:
+  home: http://proj-clhep.web.cern.ch/proj-clhep/
+  license: LGPL-3.0
+  license_family: GPL
+  license_file: CLHEP/COPYING.LESSER
+  summary: 'A Class Library for High Energy Physics'
+  description: It is intended to be a set of HEP-specific foundation and utility classes such as random generators, physics vectors, geometry and linear algebra.
```
Comment (CJ-Wright) [inline recipes/clhep/meta.yaml]: Please break up onto multiple lines

**Comment by giacomov** [comment]
@CJ-Wright I implemented your comments, thanks!

**Comment by dougalsutherland** [comment]
Merged. Thanks @giacomov, and sorry for slowness....

**Pull request merged**